src/api/java/com/minecolonies/api/colony/requestsystem/requestable/Food.java

```diff
@@ -159,18 +169,21 @@ public static Food deserialize(final IFactoryController controller, final Packet
         {
             items.add(new ItemStorage(buffer.readItem()));
         }
-
+        final int minNutrition = buffer.readInt();
         if (!items.isEmpty())
         {
-            return new Food(count, result, items);
+            return new Food(count, result, items, minNutrition);
         }
-        return new Food(count, result);
+        return new Food(count, result, minNutrition);
     }
 
     @Override
     public boolean matches(@NotNull final ItemStack stack)
     {
-        return ItemStackUtils.ISFOOD.test(stack) && !exclusionList.contains(new ItemStorage(stack)) && !(ItemStackUtils.ISCOOKABLE.test(stack) && exclusionList.contains(new ItemStorage(MinecoloniesAPIProxy.getInstance().getFurnaceRecipes().getSmeltingResult(stack))));
+        return ItemStackUtils.ISFOOD.test(stack)
+                 && !exclusionList.contains(new ItemStorage(stack))
+                 && !(ItemStackUtils.ISCOOKABLE.test(stack) && exclusionList.contains(new ItemStorage(MinecoloniesAPIProxy.getInstance().getFurnaceRecipes().getSmeltingResult(stack))))
+                 && (ItemStackUtils.ISCOOKABLE.test(stack) || stack.getItem().getFoodProperties() != null && stack.getItem().getFoodProperties().getNutrition() >= minNutrition);
```

Comment:
`ISFOOD` passing guarantees that `getFoodProperties() != null`.
What is the purpose of the `ItemStackUtils.ISCOOKABLE.test(stack) || ` check?  I don't like `&&` and `||` being combined without clarifying parentheses to indicate intended precedence, but also it doesn't seem useful, since the previous checks should have blocked anything that was cookable already.

Reply:
Also, you might want to consider making a `FoodIngredient` instance for these sorts of checks -- it's already essentially an item predicate that will match only food that has certain min/max property levels.

Reply:
The purpose is to only deliver food that is either above min or can be cooked (to end up above min)